__device-tests__/utils.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import jimp from 'jimp';
+
+const { isAndroid } = E2EUtils;
```

Comment:
I couldn't find where `E2EUtils` is defined globally, @geriux could you share where's defined? Thanks!

Reply:
Oh, I've just noticed this is defined [here](https://github.com/WordPress/gutenberg/pull/47845/files#diff-1bd8da7eecc6433e08ab95506f393935c137ac2806b7e3ec5c09796448ccb41bR7).